airbyte-integrations/connectors/source-twilio/source_twilio/streams.py

```diff
@@ -90,45 +100,53 @@ def custom_transform_function(original_value: Any, field_schema: Mapping[str, An
                 # is no need in transforming anything.
                 pass
         return original_value
+        
 
 
-class IncrementalTwilioStream(TwilioStream, ABC):
+class IncrementalTwilioStream(TwilioStream, IncrementalMixin):
     cursor_field = "date_updated"
     time_filter_template = "%Y-%m-%dT%H:%M:%SZ"
-
-    def __init__(self, start_date: str = None, **kwargs):
+    
+    def __init__(self, start_date: str = None, lookback: int = 0, **kwargs):
         super().__init__(**kwargs)
-        self._start_date = start_date
+        self._start_date = start_date  
+        self._lookback = lookback
+        self._cursor_value = None
 
     @property
     @abstractmethod
     def incremental_filter_field(self) -> str:
         """
         return: date filter query parameter name
         """
+    
+    @property
+    def state(self) -> Mapping[str, Any]:
+        if self._cursor_value:
+            return  self._cursor_value
+        else:
+            return  self._start_date
+
+    @state.setter
+    def state(self, value: Mapping[str, Any]):
+        self._cursor_value = value
```

Comment:
the state object must be a mapping. We usually declare it this way:
```suggestion
    @property
    def state(self) -> Mapping[str, Any]:
        if self._cursor_value:
            return  {self.cursor_field: self._cursor_value}
        else:
            return  {self.cursor_field: self._start_date}

    @state.setter
    def state(self, value: Mapping[str, Any]):
        self._cursor_value = max(value[self.cursor_field], self._cursor_value)
```